cli/src/constants.rs

```diff
@@ -276,6 +276,78 @@ pub const ENVS_TO_GET: &[&str] = &[
     "BUILD_TRIGGEREDBY_DEFINITIONNAME",
     "BUILD_TRIGGEREDBY_BUILDNUMBER",
     "BUILD_TRIGGEREDBY_PROJECTID",
+    //     "DRONE CI
```

Comment:
do we need to capture so many? or I suppose we could remove the ones we don't end up using later